docs/discover/set-time-filter.asciidoc

```diff
@@ -1,46 +1,53 @@
 [[set-time-filter]]
 == Setting the time filter
-You can
-set a time filter if your index contains time-based events and a time-field is
-configured for the selected index pattern.
-By default, the time filter is set to the last 15 minutes. You can use the time
-picker to change the time filter, or select a specific time 
-range in the histogram.
+If your index contains time-based events, and a time-field is configured for the 
+selected index pattern, set a time filter that displays only the data within the 
+specified time range.
 
+You can use the time filter to change the time range, or select a specific time 
+range in the histogram.
 
 [float]
-=== Filter with the time picker
+[[use-time-filter]]
+=== Use the time filter
 
-To quickly select from popular time range options, click the calendar 
-dropdown image:infrastructure/images/time-filter-calendar.png[]. 
+Use the time filter to change the time range. By default, the time filter is set 
```

Comment:
The first sentence says almost the same thing as the title.  Is it needed?  Or can you add more value?

Reply:
I didn't want the short description to be only "By default, the time filter is set to 15 minutes." This was the best I could come up with.